towers.rb

```diff
@@ -0,0 +1,234 @@
+class TowersOfHanoi
+
+  # VARIABLES AND ACCESS
+
+  attr_accessor :left_stack,
+                :center_stack,
+                :right_stack,
+                :origin_stack,
+                :destination_stack,
+                :take_from,
+                :send_to;
+
+  attr_reader :tower_height,
+              :model_stack
+
+  def valid_inputs
+    ["L","C","R"]
+  end
+
+  def stack(letter)
+    case letter
+    when "L"
+      left_stack
+    when "C"
+      center_stack
+    when "R"
+      right_stack
+    end
+  end
+
+
+  # GAME STRUCTURE
+
+  def initialize(tower_height)
+    @tower_height = tower_height
+    set_up_game
+    until game_won?
+      turn
+    end
+    game_ending_sequence
+  end
+
+  def render
+    puts
+    print "L: #{left_stack}\n"
+    print "C: #{center_stack}\n"
+    print "R: #{right_stack}\n"
+    puts
+  end
+
+  def turn
+    render
+    set_origin
+    set_destination
+    move_tile
+    reset_turn_variables
+  end
+
+  def move_tile
+    destination_stack << origin_stack.pop
+  end
+
+  def reset_turn_variables
+    self.take_from = ""
+    self.send_to = ""
+  end
+
+  # PRE-GAME SETUP
+
+  def set_up_game
+    create_model_stack
+    create_tile_stacks
+    create_movement_controls
+    game_welcome_script
+  end
+
+  def create_model_stack
+    @model_stack = (1..tower_height).to_a.reverse
+  end
+
+  def create_tile_stacks
+    @left_stack = model_stack.dup
```

Comment:
I'd declare instance variables at the top of the class so someone can easily view them all in one place